models/tokens/solana/tokens_solana_native_transfers.sql

```diff
@@ -0,0 +1,47 @@
+ {{
+  config(
+        schema = 'tokens_solana',
+        alias = 'native_transfers',
+        materialized = 'incremental',
+        file_format = 'delta',
+        incremental_strategy = 'merge',
+        incremental_predicates = [incremental_predicate('DBT_INTERNAL_DEST.block_time')],
+        unique_key = ['tx_id','outer_instruction_index','inner_instruction_index', 'block_slot'],
+        post_hook='{{ expose_spells(\'["solana"]\',
+                                    "sector",
+                                    "tokens",
+                                    \'["ilemi"]\') }}')
+}}
+--   partition_by = ['block_date'],
+
+--for the reader, note that SOL is special and can be transferred without calling the transfer instruction. It is also minted and burned without instructions. So to get balances, use daily_balances or account_activity instead of transfers.
+SELECT
+    call_block_time as block_time
+--     , date_trunc('day', call_block_time) as block_date
```

Comment:
when doing this you need to cast as date as well: `cast (date_trunc('day', call_block_time) as date)` otherwise there's sometimes problems when saving these tables.